FunctionalEquivalence/FunctionalEquivalence.wdl

```diff
@@ -0,0 +1,508 @@
+                select_all(EvalVsTruthTool1.nonSnpRocs),
+                select_all(EvalVsTruthTool2.snpRocs),
+                select_all(EvalVsTruthTool2.nonSnpRocs)
+            ])
+        
+            if (create_roc_plot[i]) {
+                call PlotROC as ROCPlot {
+                    input:
+                        rscript_http_url = roc_script_http_url,
+                        dataset_name = dataset[i],
+                        files = roc_tables_1,
+                        preemptible = preemptible
+                }
+                Array[File] roc_plots_1 = [ROCPlot.snp_plot, ROCPlot.snp_plot_zoomed, ROCPlot.snp_plot_zoomed_LCR, ROCPlot.indel_plot, ROCPlot.indel_plot_zoomed, ROCPlot.indel_plot_zoomed_LCR]
+            } # if create_roc_plot[i]
+        } # if (truth_vcf[i] != "null" && (include_in_fe_analysis[i] || create_roc_plot[i]))
+
+        if (include_in_fe_analysis[i]) {
+            call BenchmarkVCFs.Benchmark as VcfEval_Inter {
+                input:
+                    evalLabel = "tool1" + dataset[i] + replicate_no[i],
+                    truthLabel = "tool2" + dataset[i] + replicate_no[i],
+                    evalVcf = tool1_vcf[i],
+                    evalVcfIndex = tool1_vcf_index[i],
+                    truthVcf = tool2_vcf[i],
+                    truthVcfIndex = tool2_vcf_index[i],
+                    confidenceInterval = confidence_intervals[i],
+                    reference = reference,
+                    refIndex = refIndex,
+                    refDict = refDict,
+                    hapMap = hapMap,
+                    stratIntervals = stratIntervals,
+                    stratLabels = stratLabels,
+                    referenceVersion = referenceVersion,
+                    threadsVcfEval = threadsVcfEval,
+                    gatkTag = gatkTag,
+                    requireMatchingGenotypes = requireMatchingGenotypes,
+                    passingOnly = passingOnly,
+                    vcfScoreField = vcfScoreField,
+                    preemptible = preemptible,
+                    doIndelLengthStratification = false,
+                    enableRefOverlap = true
+            }
+
+            call RenameSummary as RenameSummaryInter {
+                input:
+                    input_summary = select_first([VcfEval_Inter.summary]),
+                    name = "inter" + dataset[i] + replicate_no[i],
+                    preemptible = preemptible
+            }
+
+            # Ideally we would want index j in [i+1;N] to avoid duplicates, but WDL does not support range to
+            # start at values != 0. Therefore, start at zero, count until N - i - 1 and then always add i+1.
+            #     j  in [i+1;N]     |  define j' = j-i-1
+            # <=> j' in [0;N-i-1]
+            scatter (j_minus_i_minus_1 in range(length(sample_id) - i - 1)) {
+                Int j = j_minus_i_minus_1 + i + 1
+
+                if (dataset[i] == dataset[j] && include_in_fe_analysis[j]) {
```

Comment:
Do we want to? Because the whole purpose of this is to evaluate similarity, so I don't think we'd want to make that evaluation dependent on similarity in itself. And sometimes we'd want to have different datasets of the same sample maybe (like two different coverages for example)